hefquin-engine/src/main/java/se/liu/ida/hefquin/engine/queryplan/utils/PhysicalPlanFactory.java

```diff
@@ -575,7 +584,8 @@ public static List<PhysicalPlan> enumeratePlansWithUnaryOpFromReq( final Physica
 				plans.add( createPlanWithBindJoinFILTER(gpAdd, subplan) );
 				plans.add( createPlanWithBindJoinUNION(gpAdd, subplan) );
 				//plans.add( createPlanWithBindJoinVALUES(gpAdd, subplan) );
-				plans.add( createPlanWithBindJoinVALUESorFILTER(gpAdd, subplan) );
+				//plans.add( createPlanWithBindJoinVALUESorFILTER(gpAdd, subplan) );
```

Comment:
This one should not be commented. (The reason for the previous line being commented is that the bind join with VALUES or FILTER is an actual replacement for the bind join with VALUES only.)
```suggestion
				plans.add( createPlanWithBindJoinVALUESorFILTER(gpAdd, subplan) );
```

Reply:
Ah, that makes sense!